client/src/modules/templates/bhAddItem.tmpl.html

```diff
@@ -0,0 +1,20 @@
+<div ng-form="{{ $ctrl.name }}" bh-add-item ng-model-options="{ updateOn: 'default' }">  
```

Comment:
There's no need to make an `ngForm` here ... we don't perform any form validation.  `ng-model-options` can also be removed (since it only makes sense to define on an `ngForm` or `ngModel`.